dxr/plugins/xbl/analyzer.py

```diff
@@ -0,0 +1,78 @@
+from warnings import warn
+import xml.parsers.expat as expat
```

Comment:
Expat was the first xml parser in the stdlib I found that lets me query location.